src/Drivers/Decorator.php

```diff
@@ -98,6 +100,26 @@ public function define($feature, $resolver = null): void
         });
     }
 
+    /**
+     * Discover and register the application's feature classes.
+     *
+     * @param  string  $namespace
+     * @param  string|null  $path
+     * @return void
+     */
+    public function discover($namespace = '\\App\\Features', $path = null, )
+    {
+        $namespace = Str::finish($namespace, '\\');
+
+        Collection::make((new Finder)
+                ->files()
+                ->name('*.php')
+                ->depth(0)
+                ->in($path ?? base_path('app/Features')))
+            ->each(fn ($file) => $this->define("{$namespace}{$file->getBasename('.php')}"));
+    }
+
+
```

Comment:
Put the `$namespace` first, as I felt it was probably more common to have a different app namespace than it was to have a different file structure. Once you have a different file structure, you most likely also have a different namespace.